include/UIKit/UIApplicationDelegate.h

```diff
@@ -36,17 +36,18 @@
 /* TODO 7328699::
  * Projection headers cannot currenly be included in modules, so we must forward declare classes for now
  * App developers should import the following files to use methods requiring projected classes
- * #import <UWP/WindowsFoundation.h>
- * #import <UWP/WindowsMediaSpeechRecognition.h>
+ * #import <UWP/WindowsApplicationModelActivation.h>
```

Comment:
It would be best if we could leave projection headers out of here and rely on forward-declared classes only, I think.